redhat-developer/developers.redhat.com
# DEVELOPER-5564 - Articles back to top button

### JIRA Issue Link
[DEVELOPER-5564 - Articles - Back to Top Arrow](https://issues.jboss.org/browse/DEVELOPER-5564)

- Added back the styles for the back to top button
- Added the back to top button to the article pages

### Verification Process

- Go to any article page
- Scroll down a bit
- The back to top button should appear on the lower right corner.
- When clicked, you should be taken to the top of the page
